files.sh

```diff
@@ -2,7 +2,7 @@
 
 fn nash_complete_paths(parts, line, pos) {
 	partsz   <= len($parts)
-	last     <= -expr $partsz - 1
+	last     <= -echo $partsz - 1 | bc
```

Comment:
@tiago4orion I think I found the problem: https://github.com/NeowayLabs/nash/blob/master/internal/sh/shell.go#L1650

Basically when there is an error executing the command, the variable is not set. I will open a PR with a test case for this and we can discuss a fix there, ok?